website/content/docs/usage/notebook-sample/index.md

```diff
@@ -1,99 +1,170 @@
 ---
-title: Notebook sample
+title: Kubeflow notebook sample
 weight: 10
 ---
 
-{{< warning >}}
-Incomplete. Show usage of OPA, usage of Egeria, Starting the notebook, Writing code; Also should be a KubeFlow Notebook sample.
-{{</ warning >}}
+This sample shows how to run a Kubeflow notebook with {{< name >}} and demonstrate how polices are seamlessly applied when accessing a dataset.
 
-This guides describes sample usage of {{< name >}} by describing a sample Jupyter notebook application.
-It describes how you, as a data scientist, can use a Jupyter notebook for building a fraud detection model using a 
-data asset you find in the data catalog. 
-
-# Deploy the application
+## Before you begin
+Ensure that you have the following:
+- Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
+- Kubectl
+- Kuebflow installed on your cluster (this guide was tested with Kubeflow v1.0.2)
+- S3 Object storage account (e.g., Ceph, Minio, IBM Cloud Object Storage)
 
 {{< tip >}}
-Kubernetes applications can be deployed using any standard deployment tool, from plane `kubectl apply` to [Razee](https://razee.io/).  {{< name >}} only requires a common label.
+You can install Kubeflow on Kind using ```install_kubeflow.sh``` script from {{< name >}} repository as follows:
```

Comment:
I haven't checked how it looks like in a tip box but I imagine that such text might not look nicely. You can change "using
..." to "by running [install_kubeflow.sh]" with link to the raw file and skip the bash box.